src/main/java/trn/logistics/knapsack/rest/KnapsackSolutionController.java

```diff
@@ -0,0 +1,40 @@
+package trn.logistics.knapsack.rest;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.web.bind.annotation.*;
+import trn.logistics.knapsack.dto.KnapsackSolution;
+import trn.logistics.knapsack.dto.Material;
+import trn.logistics.knapsack.dto.SolutionRequest;
+import trn.logistics.knapsack.dto.Vehicle;
+import trn.logistics.knapsack.service.KnapsackSolutionService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static trn.logistics.knapsack.rest.KnapsackSolutionController.API;
+
+
+@Slf4j
+@RestController
+@RequestMapping(API)
+public class KnapsackSolutionController {
+    public static final String API = "/api/v1/knapsackSolution";
```

Comment:
Based on [this article](https://gearheart.io/blog/restful-api-design-best-practices/#:~:text=Plural%20Nouns%20Are%20Preferable%2C%20Rather,in%20all%20good%20RESTful%20APIs.&text=Note%3A%20avoid%20mixing%20singular%20and%20plural%20nouns.) and many others, REST endpoints should be plural and not singular

This also applies to the other controllers